packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -355,6 +364,24 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   final bool obscureText;
 
+  /// {@template flutter.widgets.editableText.readOnly}
+  /// Whether the text can be changed.
+  ///
+  /// When this is set to true, the text cannot be modified
+  /// by any shortcut or keyboard operation. The text is still selectable.
+  ///
+  /// Defaults to false. Must not be null.
+  /// {@endtemplate}
+  final bool readOnly;
+
+  /// Whether to show selection handles.
+  final bool showSelectionHandles;
+
+  /// {@template flutter.widgets.editableText.showCursor}
+  /// Whether to show cursor.
```

Comment:
The sentence make sense in the constructor because the [showCursor] there refer to pass in parameter.

However, [showCursor] here will refer to the actually property Editable.showCursor. it will be a little bit confused. When parameter passed in is null, this property will actually be initialized to true or false.